### PR TITLE
Create file todolist.js to interactive functions.

### PR DESCRIPTION
**Pull request by franclobo**
### Interactive List.

- Add a new JavaScript file and import it as a module:
- It will contain methods related to the status updates (completed: true/false).
- Add event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button (use filter() method).
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/modules/todolist.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  });
+
+  trashBtnEl.addEventListener('click', () => {
+    newList.remove();
+    updateLocalStorage();
+  });
+
+  clearAll.addEventListener('click', () => {
+    if (newList.classList.contains('checked')) {
+      newList.remove();
```
Comment (FrederickMih) [inline src/modules/todolist.js]: - [x] Hi @franclobo, this actually removed all the completed tasks and the updated array of items is stored in the local storage as the requirements say. This is great :+1: . Checking more on the requirements,  we are asked to make use of the `Array.filter()` method to remove all the completed tasks from the list. Please kindly make use of this method.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @franclobo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



# To Highlights
 
- Linters are set up correctly :heavy_check_mark: 
- PR has a good title and summary :+1: 
- The updated array of items is correctly stored in the local storage :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/todolist.js (hunk 1 above)
Comment (franclobo) [reply src/modules/todolist.js]: Done.

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @franclobo , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**